src/Microsoft.AspNetCore.All/build/PublishWithAspNetCoreTargetManifest.targets

```diff
@@ -15,12 +11,21 @@ Error if PublishWithAspNetCoreTargetManifest is set to true for standalone app
 -->
   <Target
     Name="PublishWithAspNetCoreTargetManifest"
-    BeforeTargets="Publish"
-    DependsOnTargets="PrepareForPublish"
+    AfterTargets="PrepareForPublish"
     Condition="'$(PublishWithAspNetCoreTargetManifest)'=='true'" >
 
     <Error
       Text="PublishWithAspNetCoreTargetManifest cannot be set to true for standalone apps."
       Condition="'$(RuntimeIdentifier)'!='' or '$(RuntimeIdentifiers)'!=''" />
+
+    <ItemGroup>
+      <WindowsAspNetCoreTargetManifestFiles Include="$(MSBuildThisFileDirectory)aspnetcore-store-*win*.xml"/>
+      <NonWindowsAspNetCoreTargetManifestFiles Include="$(MSBuildThisFileDirectory)aspnetcore-store-*.xml" Exclude="$(MSBuildThisFileDirectory)aspnetcore-store-*win*.xml"/>
+    </ItemGroup>
+
+    <PropertyGroup>
+      <TargetManifestFiles Condition="'$(OS)' == 'Windows_NT'">$(TargetManifestFiles);@(WindowsAspNetCoreTargetManifestFiles)</TargetManifestFiles>
+      <TargetManifestFiles Condition="'$(OS)' != 'Windows_NT'">$(TargetManifestFiles);@(NonWindowsAspNetCoreTargetManifestFiles)</TargetManifestFiles>
```

Comment:
Wouldn't this mean we can no longer publish Linux apps from Windows and vice-versa?